frontend/lib/src/hooks/useScrollToBottom.ts

```diff
@@ -90,7 +90,7 @@ export function useScrollToBottom<T extends HTMLElement>(): RefObject<T> {
   }, [ignoreScrollEventBeforeRef, isAnimatingRef, setIsAnimating, setIsSticky])
 
   const handleScroll = useCallback(
-    ({ timeStampLow }) => {
+    ({ timeStampLow }: any) => {
```

Comment:
Would love to fix this type, but we can ignore for now if we need to. I imagine it's an object of type date (or number).